src/xcoobee/api/ConsentsApi.js

```diff
@@ -477,4 +518,5 @@ module.exports = {
   requestConsent,
   setUserDataResponse,
   getDataPackage,
+  registerConsents,

```

Comment:
It would be nice to get the export re-alphabetized unless there is a reason why adding to the end of the list has a benefit. I've seen many developers add to the end of a list instead of alphabetizing. So, I wonder if I am missing a benefit I am unaware of.